apps/meteor/tests/end-to-end/api/09-rooms.js

```diff
@@ -1707,4 +1707,280 @@ describe('[Rooms]', function () {
 				.end(done);
 		});
 	});
+	describe('/rooms.isMember', async () => {
```

Comment:
```suggestion
	describe('/rooms.isMember', () => {
```
I believe you don't need this async here